app/controllers/concerns/exception_handler.rb

```diff
@@ -19,7 +19,18 @@ module ExceptionHandler
       json_response({ :message => e.message }, :forbidden)
     end
 
-    rescue_from Exceptions::UserError do |e|
+    rescue_from RBACApiClient::ApiError do |e|
```

Comment:
I don't think we need to analyze any exception from an external system.

Reply:
Ok, then we will keep to return error code 500 whenever RBAC service raises exceptions.